flow/designs/gf12/bp_single/rules-base.json

```diff
@@ -42,37 +42,37 @@
         },
         {
             "field": "cts__design__instance__count__hold_buffer",
-            "value": 3703,
+            "value": 3708,
             "compare": "<="
         },
         {
             "field": "globalroute__timing__setup__ws",
-            "value": -375.46,
+            "value": -375.27,
             "compare": ">="
         },
         {
             "field": "detailedroute__route__wirelength",
-            "value": 8846424,
+            "value": 8846661,
             "compare": "<="
         },
         {
             "field": "detailedroute__route__drc_errors",
-            "value": 0,
+            "value": 1,
```

Comment:
I know this is a sensitive design, but worth noting.

Reply:
Yes.   As this flips between one and zero, I would leave the rules at 1